depecher/qml/pages/items/WritingItem.qml

```diff
@@ -556,7 +556,7 @@ Drawer {
             }
             IconButton {
                 id: sendButton
-                icon.source: "image://theme/icon-m-message"
+                icon.source: "image://theme/icon-m-sailfish"
```

Comment:
icon-m-message icon is unambiguous but sailfish icons means nothing to me. If you really want to change icon maybe it will be better to use icon-m-send similar to jolla's messages app.

Reply:
Yeah i choose icon-m-sailfish  just to see how is looking, i believe that we can use icon-m-sailfish like a intro,  I already  know that icon-m-sailfish-send is the best candidate. I just put icon-m-sailfish because i am big  fan of Sailfish, and i want to see how is work, also i am working to replace the main depecher Icon. :  )